core/crypto/crypto_store/session_keys.cpp

```diff
@@ -65,13 +65,14 @@ namespace kagome::crypto {
                                    const application::AppConfiguration &config)
       : roles_(config.roles()), store_(store) {
     if (auto dev = config.devMnemonicPhrase()) {
-      store_->generateEd25519Keypair(KEY_TYPE_GRAN, *dev).value();
-      store_->generateSr25519Keypair(KEY_TYPE_BABE, *dev).value();
-      store_->generateSr25519Keypair(KEY_TYPE_IMON, *dev).value();
-      store_->generateSr25519Keypair(KEY_TYPE_AUDI, *dev).value();
-      store_->generateSr25519Keypair(KEY_TYPE_ASGN, *dev).value();
-      store_->generateSr25519Keypair(KEY_TYPE_PARA, *dev).value();
-      store_->generateEcdsaKeypair(KEY_TYPE_BEEF, *dev).value();
+      store_->generateEd25519Keypair(KeyTypes::GRANDPA, *dev).value();
+      store_->generateSr25519Keypair(KeyTypes::BABE, *dev).value();
+      store_->generateSr25519Keypair(KeyTypes::IM_ONLINE, *dev).value();
+      store_->generateSr25519Keypair(KeyTypes::AUTHORITY_DISCOVERY, *dev)
+          .value();
+      store_->generateSr25519Keypair(KeyTypes::KEY_TYPE_ASGN, *dev).value();
+      store_->generateSr25519Keypair(KeyTypes::KEY_TYPE_PARA, *dev).value();
+      store_->generateEcdsaKeypair(KeyTypes::BEEFY, *dev).value();
```

Comment:
Can't this be done with a list and a loop?

Reply:
May make **3** loops (because of interface, each crypto type)

Reply:
There are five sr keys in a row, for example, this can be refactored to a loop. It's not necessary to make everything one loop.

Reply:
Ok, done